packages/core/src/render3/instructions/styling.ts

```diff
@@ -250,11 +251,13 @@ export function ɵɵstyleMap(styles: {[styleName: string]: any} | NO_CHANGE | nu
   // there should not be a situation where a directive host bindings function
   // evaluates the inputs (this should only happen in the template function)
   if (!isHostStyling() && hasDirectiveInput && styles !== NO_CHANGE) {
-    updateDirectiveInputValue(context, lView, tNode, bindingIndex, styles, false);
+    updateDirectiveInputValue(
+        context, lView, tNode, bindingIndex, styles, false, tView.firstUpdatePass);
```

Comment:
We should move the property access to line 240 so we don't have to read it twice (once here and once on line 257)
```suggestion
        context, lView, tNode, bindingIndex, styles, false, firstUpdatePass);
```

Reply:
fixed